homeworks/HW1/src/Courses.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Button, Form, FormGroup, Input, Col, Table, Row } from 'reactstrap';
+import { Link } from 'react-router-dom';
+import './App.css'
+export default class Courses extends React.Component {
+
+    renderCourses = () => {
+        const courses = [{
+            title: "React Redux",
+            duration: "16h",
+            description: `Adipisicing elit.
+            Harum pariatur porro quam est exercitationem rem ut corrupti
+            tempore! Mollitia ad similique dolore voluptas.
+            Corrupti saepe placeat blanditiis aliquam totam sed?`
+        },
+        {
+            title: "Node.js",
+            duration: "36h",
+            description: `Lorem ipsum dolor sit amet consectetur, adipisicing elit.
+            Nemo ratione enim laudantium aspernatur, cupiditate quis
+            placeat sapiente minus dolore ab necessitatibus aliquam magni
+            earum neque assumenda accusantium veniam est voluptatibus`
+        },
+        {
+            title: "Ajax",
+            duration: "6h",
+            description: `Nemo ratione enim laudantium aspernatur, cupiditate quis
+            placeat sapiente minus dolore ab necessitatibus aliquam magni
+            earum neque assumenda accusantium veniam est`
+        }];
+        console.log(courses)
+        return courses.map((elem, index) => {
+            return (<tr key={index}>
+                <td>
+                    <h3 className="table-course-name">{elem.title}</h3>
+                    <div className='table-course-duration'>{elem.duration}</div>
+                    {elem.description}
+                </td>
+                <td><Button className="edit-delete-course-button" type="submit" block>Edit</Button></td>
+                <td><Button className="edit-delete-course-button" type="submit" block>Delete</Button></td>
+            </tr>);
+
+        });
+    }
+
+    render() {
+        return (
+            <div>
+                <Row style={{
+                    alignItems: "center"
```

Comment:
Please don't add any CSS rules to a component. Is not a good practice. Use class Names and add styles by class.

Reply:
moved this style definition to the .css file